client/src/app/site/pages/meetings/pages/participants/pages/participant-detail/components/participant-detail-view/participant-detail-view.component.ts

```diff
@@ -80,7 +83,11 @@ export class ParticipantDetailViewComponent extends BaseMeetingComponent {
     public get shouldEnableFormControlFn(): (controlName: string) => boolean {
         return controlName => {
             const canUpdateUsers = this.isAllowed(`update`);
-            if (this._isUserInScope || (this.newUser && canUpdateUsers)) {
+            if (this.newUser && canUpdateUsers) {
+                return true;
+            } else if (this._isUserEditable && controlName !== `default_password`) {
+                return true;
+            } else if (this._isUserIniPWEditable && controlName === `default_password`) {
```

Comment:
Just write it out and also do not introduce a new naming when there is an existing one. 
```suggestion
            } else if (this._isUserDefaultPasswordEditable && controlName === `default_password`) {
```
Or maybe even
```suggestion
            } else if (this._isDefaultPasswordEditable && controlName === `default_password`) {
```

Reply:
Renamed into second version.